docs/_includes/common/details.html

```diff
@@ -0,0 +1,143 @@
+<div id="details">
+    <h2><span class="secondary-text">@ebay/skin/</span>details</h2>
+    <p>A details element is an interactive element used to show and hide content.</p>
+    <p>It uses the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details">details</a> and <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary">summary</a> disclosure elements to show/hide content.</p>
+    <p>Everything inside <span class="highlight">details</span> and below <span class="highlight">summary</span> is considered the content.</p>
+    <p>Since IE/Edge do not support details and summary elements. The <span class="highlight">details-element-polyfill</span> is used to solve this problem.</p>
+    <h3>Default Details Summary</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <details class="details">
+                <summary class="details__summary">
+                    <span class="details__summary--text">Details</span><span class="details__icon"></span>
```

Comment:
@szhou1 You have some invalid BEM here. Double dash denotes a modifier to an existing block class on the same element. Given that there is no exisiting block class on this element, I think you probably meant for this to be a single dash, `details__summary-text`.

Reply:
Also, `details__summary-text` and `details__icon` on the same element looks a little odd. Maybe give them both the word summary, so: `details__summary-text` and `detail__summary-icon`.

Reply:
ok i'll fix that